package.json

```diff
@@ -1,25 +1,29 @@
 {
   "private": true,
   "scripts": {
+    "reinstall": "rimraf ./**/node_modules && yarn install",
     "prepare": "lerna run prepare",
-    "rebuild:browser": "theia rebuild:browser",
-    "rebuild:electron": "theia rebuild:electron",
-    "start:browser": "yarn rebuild:browser ; yarn --cwd browser-app start",
-    "start:electron": "yarn rebuild:electron ; yarn --cwd electron-app start",
+    "build": "lerna run build",
+    "build:browser": "theia rebuild:browser",
```

Comment:
I would advise to keep the original names: `rebuild:*` - it's kind of a convention and does not build the app, just rebuilds the native npm dependencies for either `browser` or `electron`.